package.json

```diff
@@ -3,9 +3,10 @@
   "version": "1.0.2",
   "description": "Sophisticated and functionally-minded async with advanced features: coroutines, promises, ES2015 iterables, fantasy-land",
   "main": "dist/creed.js",
-  "jsnext:main": "src/main.js",
+  "jsnext:main": "dist/creed.es.js",
```

Comment:
If we did include `jsnext:main` with more finely grained modules, I guess one way to do that would be to point it at `build/src/main.js`, and include the `build/src` dir in the npm package.  That seems pretty reasonable.  Is that what you were thinking, @bergus?

> I guess it doesn't make a huge difference here.

I really don't know, tbh.  My intuition is that the public API is the same in both cases, which is what seems like it would ultimately drive the tree shaking.